test/config/benchmark_config.py

```diff
@@ -0,0 +1,41 @@
+from settings import *
+
+ANSIBLE_AUTH_FILE = '.ansible_auth'
+
+# Obfuscate credentials
+def set_ansible_auth(local_user, local_pwd, host_ssh_port, host_ssh_user, host_ssh_pwd):
```

Comment:
You could reuse the settings.py b64 methods:
https://github.com/RackHD/RackHD/blob/master/test/config/settings.py#L26

Maybe rename them to something generic like `set_b64_creds()`.